target/linux/ath79/image/generic.mk

```diff
@@ -251,6 +251,24 @@ define Device/buffalo_wzr-hp-ag300h
 endef
 TARGET_DEVICES += buffalo_wzr-hp-ag300h
 
+define Device/buffalo_wzr-hp-g300nh
+  SOC := ar9132
+  DEVICE_VENDOR := Buffalo
+  DEVICE_MODEL := WZR-HP-G300NH
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ledtrig-usbport kmod-gpio-nxp-74hc153
+  BLOCKSIZE := 128k
+  IMAGE_SIZE := 32128k
+  IMAGES += factory.bin tftp.bin
+  IMAGE/default := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | \
+	pad-rootfs | check-size
+  IMAGE/factory.bin := $$(IMAGE/default) | buffalo-enc WZR-HP-G300NH 1.99 | \
+	buffalo-tag WZR-HP-G300NH 1
+  IMAGE/sysupgrade.bin := $$(IMAGE/default) | append-metadata
```

Comment:
That's nothing different than what's in Device/Default (except for the order of the last two parameter, so I'd drop this line.